src/main/java/frc/robot/subsystems/tank/TankSubsystem.java

```diff
@@ -1,31 +1,31 @@
-package frc.robot.subsystems;
+package frc.robot.subsystems.tank;
 
-import com.kauailabs.navx.frc.AHRS;
+import static frc.robot.Constants.TankConstants.bLeftMotorPort;
```

Comment:
Why this instead of the previous 
```java
import static frc.robot.Constants.TankConstants.*;
```
?

Reply:
auto format

Reply:
Importing with a `*` is a bad idea. You don't know all that you're bringing in to the namespace. Better to be explicit.